reference/docs-conceptual/components/ise/How-to-Debug-Scripts-in-Windows-PowerShell-ISE.md

```diff
@@ -1,31 +1,50 @@
+A breakpoint is a designated spot in a script where you would like operation to pause so that you
+can examine the current state of the variables and the environment in which your script is running.
+Once your script is paused by a breakpoint, you can run commands in the Console Pane to examine the
+state of your script. You can output variables or run other commands. You can even modify the value
+of any variables that are visible to the context of the currently running script. After you have
+examined what you want to see, you can resume operation of the script.
 
 You can set three types of breakpoints in the Windows PowerShell debugging environment:
 
-1. **Line breakpoint**. The script pauses when the designated line is reached during the operation of the script
+1. **Line breakpoint**. The script pauses when the designated line is reached during the operation
+   of the script
 
-2. **Variable breakpoint.** The script pauses whenever the designated variable's value changes.
+1. **Variable breakpoint.** The script pauses whenever the designated variable's value changes.
 
-3. **Command breakpoint.** The script pauses whenever the designated command is about to be run during the operation of the script. It can include parameters to further filter the breakpoint to only the operation you want. The command can also be a function you created.
+1. **Command breakpoint.** The script pauses whenever the designated command is about to be run
+   during the operation of the script. It can include parameters to further filter the breakpoint to
+   only the operation you want. The command can also be a function you created.
 
-Of these, in the Windows PowerShell ISE debugging environment, only line breakpoints can be set by using the menu or the keyboard shortcuts. The other two types of breakpoints can be set, but they are set from the Console Pane by using the [Set-PSBreakpoint](https://technet.microsoft.com/library/88d2d9ad-17dc-44ae-99aa-f841125b9dc8) cmdlet. This section describes how you can perform debugging tasks in Windows PowerShell ISE by using the menus where available, and perform a wider range of commands from the Console Pane by using scripting.
+Of these, in the Windows PowerShell ISE debugging environment, only line breakpoints can be set by
+using the menu or the keyboard shortcuts. The other two types of breakpoints can be set, but they
+are set from the Console Pane by using the [Set-PSBreakpoint](https://technet.microsoft.com/library/88d2d9ad-17dc-44ae-99aa-f841125b9dc8)
```

Comment:
These old links for cmdlets need to be updated throughout each of the updated files.

Reply:
forgot all about these. Thanks

Reply:
Fixed